### DISCUSSION TITLE
Blank space shows when no announcment is active

### DISCUSSION THREAD

**Discussion by SilentGlasses** [squidfunk/mkdocs-material#4169] [Q&A] [answered]
I have the announcement feature enabled on my site, however, when no announcement is set, there is this bank of space that shows on the top of the page . Is there a way to get rid of it?

my `overrides/main.html` file:

```
{% extends "base.html" %}

<!-- ANNOUNCMENTS -->
{% block announce %}

{% endblock %}
```

![Screen Shot 366](https://user-images.githubusercontent.com/22822565/181056610-ea033ae9-4471-41bd-a1e2-c6dfb41490f7.png)

**Comment by squidfunk** [comment, marked as answer]
The reason is that you have blank spaces in your template:

```
{% block announce %}

{% endblock %}
```

Try either this:

```
{% block announce %}{% endblock %}
```

Or this:

```
{% block announce -%}

{%- endblock %}
```

For further reference see [whitespace control in Jinja](https://jinja.palletsprojects.com/en/3.0.x/templates/#whitespace-control).

**Comment by SilentGlasses** [comment]
Forgot to get back to you. After a recent update, the issue went away. I have also switched it to the second option you suggested to be safe...

```
{% block announce -%}

{%- endblock %}
```
